### DISCUSSION TITLE
rocketmq topic perm的值在什么情况下会被修改

### DISCUSSION THREAD

**Discussion by LittleBoy18** [apache/rocketmq#7595] [Q&A]
环境：在openshift环境中通过pod部署，使用了类似nfs的远程存储
版本：5.1.0

反复重启broker pod，发生了broker重新调度到其他节点，会出现topic perm值被修改为4的情况

**Comment by francisoliverlee** [comment]
相同问题，5.3版本也遇到了。通过重启namesrv解决了。namesrv上有3处地方修改，貌似都不是我的情况，有进度分享

<img width="978" height="773" alt="Clipboard_Screenshot_1764658999" src="https://github.com/user-attachments/assets/713e0af2-bd16-44f6-bab1-c12d57a89604" />

<img width="1021" height="613" alt="Clipboard_Screenshot_1764659011" src="https://github.com/user-attachments/assets/2a0bc0e0-78cf-41fb-ae97-d994ebeb6adf" />
